src/search/views.py

```diff
@@ -38,14 +38,14 @@ def search(request):
         query_string = None
         feeds = []
         from_feed = None
-    # �ŏI�X�V����
+    # �ŏI�X�V����
     last_reload = str(datetime.datetime.now())
-    # anonymous�ȊO�̑S���[�U��ԋp����
+    # anonymous�ȊO�̑S���[�U��ԋp����
```

Comment:
Is the character code okay?

Reply:
Thank you for your comment.
I will remove comment lines.